shared/src/main/scala/io/kaitai/struct/languages/AwkwardCompiler.scala

```diff
@@ -744,10 +775,8 @@ class AwkwardCompiler(
           }
           outSrc.puts(s"${builderName}_listoffsetbuilder.begin_list();")
           outSrc.puts(s"auto& ${idToStr(id)}_builder = ${builderName}_listoffsetbuilder.content();")
-          if(dataType.isInstanceOf[StrType]) {
-            outSrc.puts(s"""${builderName}_listoffsetbuilder.set_parameters("\\"__array__\\": \\"string\\"");""")
-            outSrc.puts(s"""${idToStr(id)}_builder.set_parameters("\\"__array__\\" : \\"char\\"");""")
-          }
+          outSrc.puts(s"""${builderName}_listoffsetbuilder.set_parameters("\\"__array__\\": \\"string\\"");""")
+          outSrc.puts(s"""${idToStr(id)}_builder.set_parameters("\\"__array__\\" : \\"char\\"");""")
```

Comment:
For bytes, parameters will be `bytestring` instead of `string` and `byte` instead of `char`.

Reply:
is it possible that it is the default behavior and I do not need to specify that?

Reply:
By default, the `parameters` are empty so it needs to be specified. I had missed it earlier for `BytesType`.